codebuild/cd/test_version_exists.yml

```diff
@@ -0,0 +1,19 @@
+version: 0.2
+#this build spec assumes the ubuntu 14.04 trusty image
```

Comment:
Might be worth a comment explaining how this is a canary used to gate the other builds